test/LocalNeighborhood/test_ln.py

```diff
@@ -12,7 +12,8 @@
 # Modify the path because of the - in the directory
 SPRAS_ROOT = Path(__file__).parent.parent.parent.absolute()
 sys.path.append(str(Path(SPRAS_ROOT, 'docker-wrappers', 'LocalNeighborhood')))
-from local_neighborhood import local_neighborhood
+from local_neighborhood import local_neighborhood 
```

Comment:
Right - you need to remove this little snippet here, then you can uncomment the `import LocalNeighborhood as LocalNeighborhoodWrapper`